osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderRepeat.cs

```diff
@@ -57,8 +58,12 @@ private void load()
                 Children = new Drawable[]
                 {
                     // no default for this; only visible in legacy skins.
-                    CirclePiece = new SkinnableDrawable(new OsuSkinComponent(OsuSkinComponents.SliderTailHitCircle), _ => Empty()),
-                    arrow = new ReverseArrowPiece(),
+                    CirclePiece = new SkinnableDrawable(new OsuSkinComponent(OsuSkinComponents.SliderTailHitCircle), _ => Empty())
+                    {
+                        Anchor = Anchor.Centre,
+                        Origin = Anchor.Centre,
```

Comment:
Feel like these should probably be applied at `SkinnableDrawable` itself, using the existing `CentreComponent` flag, but don't want to make that change here.